go.mod

```diff
@@ -9,6 +9,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/jonboulle/clockwork v0.4.0 // indirect
```

Comment:
Added this external dependency, but it's used only in tests. Would be nice to implement our own fake clock. At least there are no other indirect dependencies, the clockwork library is small.